.github/workflows/publish-image.yaml

```diff
@@ -34,3 +34,4 @@ jobs:
             ghcr.io/${{ github.repository }}/powerdns:${{ steps.tag.outputs.bare }}
             ghcr.io/${{ github.repository }}/powerdns:${{ steps.tag.outputs.full }}
           cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
Why to not add `cache-to` to all build steps and just let the action decide if it has a layer that needs to be cached or not?

I am talking also about the other builds at PR time.

Reply:
Mostly to save the 5 secs that takes pushing those to GH